### ISSUE TITLE
assign style to table by row?

### ISSUE THREAD

**Issue by retsyo** [python-openxml/python-docx#370]
currently, I do this by something like
```python
head = ['Name', 'Birthday', 'Phone No']
for i in range(len(head)):
    tab.cell(0, i).text = head[i]
    for par in tab.cell(0, i).paragraphs:
        par.style = 'my style'
```

is it possible to do something like?
```python
head = ['Name', 'Birthday', 'Phone No']
for i in range(len(head)):
    tab.cell(0, i).text = head[i]
    
tab.rows[0].style = 'my style'
```

Thnaks

**Comment by davidasboth** [comment]
@retsyo did you get this to work?

I have a similar question. @scanny what are our options for conditionally changing row styles (e.g. background colour)? Or are they set at the table-level? In which case, is there a way to access background colour etc. of a table cell/row and change it dynamically?

**Comment by scanny** [comment]
@davidasboth there are no "row" styles, just a table style. If you want to do something like highlight a row you need to do it for each cell in that row individually.

**Comment by davidasboth** [comment]
Thanks, yeah I wasn't convinced you could have separate styles for separate parts of a table. I couldn't find the right interface for changing a cell's properties like background color, where should I be looking @scanny?

**Comment by scanny** [comment]
I don't believe any of that is implemented yet so you'd be on your own to develop or find someone who's worked out a workaround function that manipulates the underlying XML directly. I'd start with a Google of "python-docx cell shading" and see where that takes you.

**Comment by davidasboth** [comment]
I figured as much. I'll see what I can do with the XML. Thanks again for the quick replies.

**Comment by davidasboth** [comment]
The first result already looks promising (to give future readers a head start): https://stackoverflow.com/questions/26752856/python-docx-set-table-cell-background-and-text-color
